a3_finance/a3_finance/doctype/lop_per_request/lop_per_request.py

```
@@ -10,7 +10,7 @@ class LopPerRequest(Document):
 	def validate(self):
 		from frappe.utils import getdate
```

Ruff (0.12.2)

Redefinition of unused `getdate` from line 7

Remove definition: `getdate`

(F811)